manifests/o/OpenMW/OpenMW/0.47.0/OpenMW.OpenMW.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: OpenMW.OpenMW
+PackageVersion: 0.47.0
+PackageLocale: en-US
+Publisher: OpenMW Council

```

```suggestion
Publisher: OpenMW.org
```
Must match Apps&Features / Control Panel